### PR TITLE
TastyCloud documentation

### PR DESCRIPTION
**Pull request by ReynardtDeminey**

### CODE REVIEW

File: content/apps/tastycloud/en/connect-hubrise.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Connect to HubRise
+position: 2
+layout: documentation
+meta:
+  title: Tasty Cloud Connection to HubRise
+  description: Instructions on connecting Tasty Cloud with HubRise.
+---
+
+TastyCloud provides management of multiple branches from a single back office. These branches can be connected to a HubRise Location, sharing menu, orders, and customer information with HubRise.
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/connect-hubrise.md:10]: I changed the wording here a bit compared to the original Google Docs version.

File: content/apps/tastycloud/en/connect-hubrise.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+The instructions below detail how to connect TastyCloud to an existing HubRise Account.  For information on how to create your HubRise user profile and Account, see the [HubRise User Guide](https://www.hubrise.com/apps/hubrise/).
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/connect-hubrise.md:12]: I changed the wording a bit in respect of the connection in order to improve the readability and structure.

File: content/apps/tastycloud/en/faqs.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: FAQs
+position: 6
+layout: documentation
+meta:
+  title: TastyCloud Connection to HubRise FAQs
+  description: Frequently asked questions and their answers.
+---
+
+No frequently asked questions are available at this time.
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/faqs.md:10]: As in Zelty, I just mentioned that there are no frequently asked question at this time.

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+---
+title: Map Ref Codes
+position: 3
+layout: documentation
+meta:
+  title: TastyCloud Connection to HubRise Configuration
+  description: Instructions on how to configure TastyCloud for optimal connection to HubRise and other platforms.
+---
+
+Once your TastyCloud site is connected with a HubRise account, customer and order information is transmitted to HubRise automatically.  To ensure the information is sent accurately, some configuration on the TastyCloud side is required.  This information is especially true for the connection between your TastyCloud Site and the EPOS connected to your HubRise Account.
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/map-ref-codes.md:10]: I converted everything from the original Google Docs to markdown. I added the paragraphs at the end.

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+1. Login to your TastyCloud back office.
+1. Click **Mapping des produits** from the left navigation panel.
+1. Select from the following options depending on which mapping you are updating:
+    - Produits
+    - Options
+    - Quantité
+1. From the row with the TastyCloud and HubRise mapping, click the **Delete icon** on the right hand side.
+
+## Deals and Discounts
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/map-ref-codes.md:64]: Added this

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+TastyCloud offers deals by way of flexible pricing and discounts which can be applied to an order.  Correct EPOS ref codes are essential for orders containing discounts to be properly transmitted to your EPOS system and other apps connected to HubRise. 
+
+To create a new discount:
+
+1. Select **Promotions** from the left navigation pane.
+1. Fill in all the necessary details.
+1. Click **Enregister**.
+
+
+## Payment Methods
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/map-ref-codes.md:75]: Added this

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+TastyCloud offers various payment methods, including cash and online payment. No ref codes are specified for the different methods and TastyCloud only sends the type of payment through to HubRise.
+
+## Service Types
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/map-ref-codes.md:79]: Added this

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+Service Type such as Delivery, Collection or Eat in are supported. These service types are not supported by unique ref codes in TastyCloud, but might require a EPOS code entry. Refer to your connected EPOS documentation on the HubRise website to verify.
+
+Service Types may be added in your TastyCloud back office.
+
+## Charges
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/map-ref-codes.md:85]: Added this

File: content/apps/tastycloud/en/overview.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/overview.md:1]: Changed the wording a bit compared to the original Google Docs version

File: content/apps/tastycloud/en/troubleshooting.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+    - Options
+    - Quantité
+1. The mapping page displays a table showing the current mappings between TastyCloud and HubRise.  The left side displays the Id and the Name of the TastyCloud product, option, or quantity.  The right side displays the HubRise Id and Name that the production, option or quantity is mapped to.
+1. The Produits dropdown menu includes all TastyCloud products, options or quantities that have not been mapped to Hubrise.
+
+Should any products not be mapped, you can proceed to [map](/apps/tastycloud/map-ref-codes) them.
+
+## TastyCloud Support
+
+The TastyCloud support team can be contacted at _________@tastycloud.fr for issues with the configuration of the online ordering system.
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/troubleshooting.md:33]: Could not find the email address for support

File: content/apps/tastycloud/en/troubleshooting.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/troubleshooting.md:1]: As no troubleshooting was contained in the original Google Docs, I added my own. I hope this is right

File: content/apps/tastycloud/en/connect-hubrise.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+meta:
+  title: Tasty Cloud Connection to HubRise
+  description: Instructions on connecting Tasty Cloud with HubRise.
+---
+
+TastyCloud provides management of multiple branches from a single back office. These branches can be connected to a HubRise Location, sharing menu, orders, and customer information with HubRise.
+
+The instructions below detail how to connect TastyCloud to an existing HubRise Account.  For information on how to create your HubRise user profile and Account, see the [HubRise User Guide](https://www.hubrise.com/apps/hubrise/). 
+
+In order to connect your HubRise account to your TastyCloud site, you should contact your TastyCloud support representative. They will assist you with the integration of your HubRise Account to your TastyCloud site.
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/connect-hubrise.md:14]: Original comment by John: These instructions are based on the current build. I believe there is an automated version but I haven't seen that yet.

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+## Catalog Import
+
+### Display Mapping from TastyCloud Menu to HubRise Catalog
+
+To map the TastyCloud Menu to the HubRise Catalog, a connection between TastyCloud and HubRise must be established.  For more information on connecting these two systems, see [Connect TastyCloud with HubRise](/apps/tastycloud/connect-hubrise).
+
+Once the connection is established, follow these steps to display the mapping between your TastyCloud menu with the catalog:
+
+1. Login to your TastyCloud back office.
+1. Click **Mapping des produits** from the left navigation panel.
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/map-ref-codes.md:23]: Original comment by John: Keeping the UI interface options for now.

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+    - Options
+    - Quantité
+1. The mapping page displays a table showing the current mappings between TastyCloud and HubRise.  The left side displays the Id and the Name of the TastyCloud product, option, or quantity.  The right side displays the HubRise Id and Name that the production, option or quantity is mapped to.
+1. The Produits dropdown menu includes all TastyCloud products, options or quantities that have not been mapped to Hubrise. 
+
+### Add TastyCloud Menu to HubRise Catalog Mapping
+
+To add a new mapping between a TastyCloud Menu product, option or quantity to HubRise:
+
+1. Login to your TastyCloud back office.
```
Comment (ReynardtDeminey) [inline content/apps/tastycloud/en/map-ref-codes.md:35]: Original comment by John: Based on this video: https://youtu.be/r8Wz_hK3I1I

File: content/apps/tastycloud/en/connect-hubrise.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+
+TastyCloud provides the management of multiple branches from a single back office. These branches can be connected to a HubRise Location, which enables the sharing of menu, orders, and customer information with HubRise.
+
+The instructions below detail how to connect TastyCloud to an existing HubRise Account.  For information on how to create your HubRise user profile and Account, see the [HubRise User Guide](https://www.hubrise.com/apps/hubrise/). 
+
+In order to connect your HubRise account to your TastyCloud site, you should contact your TastyCloud support representative. They will assist you with the integration of your HubRise Account to your TastyCloud site.
+
+---
+
+**IMPORTANT NOTE**: At this stage, the integration of a HubRise Account to a TastyCloud site can only be performed by TastyCloud's support team, but an automated process for all users will be available soon.
```
Comment (janainawittner) [inline content/apps/tastycloud/en/connect-hubrise.md:18]: To be confirmed.

File: content/apps/tastycloud/en/connect-hubrise.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+meta:
+  title: Tasty Cloud Connection to HubRise
+  description: Instructions on connecting Tasty Cloud with HubRise.
+---
+
+TastyCloud provides the management of multiple branches from a single back office. These branches can be connected to a HubRise Location, which enables the sharing of menu, orders, and customer information with HubRise.
+
+The instructions below detail how to connect TastyCloud to an existing HubRise Account.  For information on how to create your HubRise user profile and Account, see the [HubRise User Guide](https://www.hubrise.com/apps/hubrise/). 
+
+In order to connect your HubRise account to your TastyCloud site, you should contact your TastyCloud support representative. They will assist you with the integration of your HubRise Account to your TastyCloud site.
```
Comment (janainawittner) [inline content/apps/tastycloud/en/connect-hubrise.md:14]: TastyCloud will need a HubRise token to manually enter it in the Admin > POS page of the TastyCloud back office. I am not sure what this token is. We need to investigate.

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 17]
```diff
@@ -0,0 +1,86 @@
...
+meta:
+  title: TastyCloud Connection to HubRise Configuration
+  description: Instructions on how to configure TastyCloud for optimal connection to HubRise and other platforms.
+---
+
+Once your TastyCloud site is connected with a HubRise account, customer and order information is transmitted to HubRise automatically. To ensure the information is sent accurately, some configuration on the TastyCloud side is required. This is especially true for the connection between your TastyCloud site and the EPOS connected to your HubRise Account.
+
+Both TastyCloud and HubRise store the items that are sold to customers. The mapping between these two systems is managed by the TastyCloud back office system.
+
+## Catalog Import
```
Comment (janainawittner) [inline content/apps/tastycloud/en/map-ref-codes.md:14]: From my understanding of Ernest's presentation, TastyCloud automatically pulls Catalogs from HubRise when there is an update. Nothing needs to be done for this to happen, the an automatic import.

Unless the synch is done when you click on Produit.

 To be confirmed.

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 18]
```diff
@@ -0,0 +1,71 @@
...
+
+## Deals and Discounts
+
+TastyCloud offers deals by way of flexible pricing and discounts which can be applied to an order. Correct EPOS ref codes are essential for orders containing discounts to be properly transmitted to your EPOS system and other apps connected to HubRise.
+
+To create a new discount:
+
+1. Select **Promotions** from the left navigation pane.
+1. Fill in all the necessary details.
+1. Click **Enregister**.
```
Comment (janainawittner) [inline content/apps/tastycloud/en/map-ref-codes.md:59]: Not sure how deals and discounts are mapped.

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 19]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+**IMPORTANT NOTE**: TastyCould pulls the menu from HubRise catalogs. Your EPOS generally generates these catalogs by pushing product items into it. Refer to your connected EPOS documentation on the HubRise website to verify that your EPOS supports this integration feature.
+
+---
+
+Follow these steps to map items between a TastyCloud and HubRise:
+
+1. Login to your TastyCloud back office.
+1. Click **Mapping des produits** (Products mapping) from the left navigation panel.
```
Comment (janainawittner) [inline content/apps/tastycloud/en/map-ref-codes.md:31]: What does **Menus** option correspond to in this context?

File: content/apps/tastycloud/en/map-ref-codes.md [hunk 20]
```diff
@@ -0,0 +1,71 @@
...
+
+## Product
+
+Both TastyCloud and HubRise store the items that are sold to customers. The mapping between items on these two systems is managed by the TastyCloud back office.
+
+To map the TastyCloud menu to the HubRise Catalog, a connection between TastyCloud and HubRise must be established. For more information on connecting these two systems, see [Connect TastyCloud to HubRise](/apps/tastycloud/connect-hubrise).
+
+---
+
+**IMPORTANT NOTE**: TastyCould pulls the menu from HubRise catalogs. Your EPOS generally generates these catalogs by pushing product items into it. Refer to your connected EPOS documentation on the HubRise website to verify that your EPOS supports this integration feature.
```
Comment (janainawittner) [inline content/apps/tastycloud/en/map-ref-codes.md:24]: What happens if TastyCloud is connected to an EPOS that does not pull the menu? 
Is TastyCloud planning to update their system to allow a manual mapping by ref code entry in their back office?

**Comment by janainawittner** [comment]
@antmonnier, I have reviewed it and done as much as I can. 

There are pending questions we will need to ask TastyCloud. Perhaps you can answer some of them by looking at the integration sheet. See unreslved conversations here above. I saw the whole video again but it does not tell me much...

File: content/apps/tastycloud/en/map-ref-codes.md (hunk 18 above)
Comment (janainawittner) [reply content/apps/tastycloud/en/map-ref-codes.md:59]: In the video, you speak with bilal about it at 44min 25 secs
https://drive.google.com/file/d/1PiBvE4NvqzwhebCLqXCspwBYbgTkhnq8/view?usp=sharing

File: content/apps/tastycloud/en/tastycloud-terms.md [hunk 21]
```diff
@@ -0,0 +1,13 @@
+---
+title: TastyCloud Terms
```
Comment (janainawittner) [inline content/apps/tastycloud/en/tastycloud-terms.md:2]: Would it be fair to include here **Formule** (Bundled deal) on TastyCloud = Deal in HubRise.

File: content/apps/tastycloud/en/connect-hubrise.md [hunk 22]
```diff
@@ -0,0 +1,28 @@
...
+
+The instructions below detail how to connect TastyCloud to an existing HubRise Account.
+
+1. Create your HubRise user profile and Account. For instructions, see [Create a User Profile](/docs/getting-started/#create-a-user-profile).
+1. Give support@tastycloud.fr permission to access your HubRise Account. For instructions, see [Add a User Permission](/docs/permissions/#add-a-user).
+1. Contact your TastyCloud support representative and share your HubRise Account name and unique identifier. They will assist you with the integration of your HubRise Account to your TastyCloud site.
+
+---
+
+**IMPORTANT NOTE**: At this stage, the integration of a HubRise Account to a TastyCloud site can only be performed by TastyCloud's support team, but soon users will be able to do it autonomously.
```
Comment (janainawittner) [inline content/apps/tastycloud/en/connect-hubrise.md:20]: Is it true that users will soon be able to do it autonomoulsy?

File: content/apps/tastycloud/en/connect-hubrise.md [hunk 23]
```diff
@@ -0,0 +1,28 @@
+---
+title: Connect to HubRise
```
Comment (janainawittner) [inline content/apps/tastycloud/en/connect-hubrise.md:2]: How do we send test orders?

File: content/apps/tastycloud/en/connect-hubrise.md (hunk 22 above)
Comment (antmonnier) [reply content/apps/tastycloud/en/connect-hubrise.md:20]: That's what is written in the integration sheet, so probably correct.

File: content/apps/tastycloud/en/connect-hubrise.md (hunk 16 above)
Comment (antmonnier) [reply content/apps/tastycloud/en/connect-hubrise.md]: I'm not sure we want to document this: I cannot see the "POS > Ajouter un restaurant" in our Test account, and therefore we cannot guarantee this process is and will remain accurate.

File: content/apps/tastycloud/en/map-ref-codes.md (hunk 20 above)
Comment (antmonnier) [reply content/apps/tastycloud/en/map-ref-codes.md]: Good point, will report in the sheet.

File: content/apps/tastycloud/en/tastycloud-terms.md (hunk 21 above)
Comment (antmonnier) [reply content/apps/tastycloud/en/tastycloud-terms.md:2]: Done (it's also called Menu)

File: content/apps/tastycloud/en/map-ref-codes.md (hunk 19 above)
Comment (antmonnier) [reply content/apps/tastycloud/en/map-ref-codes.md:28]: It's explained in the Deals section below.

File: content/apps/tastycloud/en/map-ref-codes.md (hunk 18 above)
Comment (antmonnier) [reply content/apps/tastycloud/en/map-ref-codes.md]: Done, documented.

File: content/apps/tastycloud/en/connect-hubrise.md (hunk 22 above)
[the remark this answers is not in the document]
Comment (antmonnier) [reply content/apps/tastycloud/en/connect-hubrise.md]: TastyCloud requires dual menu entry. The HubRise catalog is only used to populate the ref codes, using a manual matching process.

File: content/apps/tastycloud/en/connect-hubrise.md (hunk 23 above)
Comment (antmonnier) [reply content/apps/tastycloud/en/connect-hubrise.md:2]: Why do you feel this should be documented?

File: content/apps/tastycloud/en/connect-hubrise.md (hunk 16 above)
Comment (janainawittner) [reply content/apps/tastycloud/en/connect-hubrise.md]: Better ask in the integration sheet then. 
I think we need to keep this info somewhere for us to have a better understanding of how things are done. In the internal wiki perhaps?

File: content/apps/tastycloud/en/map-ref-codes.md (hunk 17 above)
Comment (janainawittner) [reply content/apps/tastycloud/en/map-ref-codes.md]: OK. If I undertand, queries here have been transferred to the IntegrationSheet. So I think this conversation can be resoled. You do it?

File: content/apps/tastycloud/en/map-ref-codes.md (hunk 20 above)
Comment (janainawittner) [reply content/apps/tastycloud/en/map-ref-codes.md]: I let you **Resolve this conversation** as soon as the question has been asked in the IntegrationSheet.

File: content/apps/tastycloud/en/connect-hubrise.md (hunk 22 above)
[the remark this answers is not in the document]
Comment (janainawittner) [reply content/apps/tastycloud/en/connect-hubrise.md]: Yes, definitely. Dual menu entry. Sort of manual code ref mapping, without the need to cut/paste. Not idea, but better than nothing.

If this was not clear in the documentation, it should be highlighted once more.

I let you **Resolve this conversation** when you are done.

Comment (janainawittner) [reply content/apps/tastycloud/en/connect-hubrise.md]: I let you **Resolve this conversation** as soon as the question has been asked in the IntegrationSheet.

File: content/apps/tastycloud/en/connect-hubrise.md (hunk 23 above)
Comment (janainawittner) [reply content/apps/tastycloud/en/connect-hubrise.md:2]: No idea why I asked this question. Just Resolved this convesation.

**Comment by janainawittner** [comment]
@antmonnier, I answered all your questions above. I let you **Resolve conversation** as needed. Let me know if you need anything else.

File: content/apps/tastycloud/en/connect-hubrise.md (hunk 16 above)
Comment (antmonnier) [reply content/apps/tastycloud/en/connect-hubrise.md]: Anything not "secret" (private email addresses, private links, etc.) should be public. I'm including the process in the doc.

Comment (janainawittner) [reply content/apps/tastycloud/en/connect-hubrise.md]: OK. Even better.

**Pull request merged**